.github/workflows/dev-short-tests.yml

```diff
@@ -178,6 +178,51 @@ jobs:
           make clean
           CC=clang MOREFLAGS="-Werror -Wimplicit-fallthrough -O0" make -C lib -j libzstd.a ZSTD_LEGACY_SUPPORT=0
 
+  meson-linux:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install packages
+        run: |
+          sudo apt-get update
+          sudo apt-get -y install build-essential python3-pip ninja-build liblz4-dev
+          pip install --pre meson
+      - name: Build with Meson
+        run: |
+          meson setup \
+            --buildtype=debugoptimized \
+            -Db_lundef=false \
+            -Dauto_features=enabled \
+            -Dbin_programs=true \
+            -Dbin_tests=true \
+            -Dbin_contrib=true \
+            -Ddefault_library=both \
+            build/meson builddir
+          ninja -C builddir/
+          meson test -C builddir/ --print-errorlogs
+          meson install -C builddir --destdir staging/
+
+  meson-windows:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install packages
+        run: pip install --pre meson
+      - name: Initialize the MSVC dev command prompt
+        uses: ilammy/msvc-dev-cmd@v1
```

Comment:
We should specify a commit hash rather than a tag, especially when depending on third-party actions.

Reply:
Fair enough. :) I tend to avoid third-party ones too, at least for workflows that have secrets! but I don't usually bother with hash pinning since I either trust the action source or don't give it secrets -- and I copied this from a workflow that doesn't have secrets.

I'll make this change when rebasing (soon).

Reply:
Yeah, it's probably safe, but at the same time it's a better practice to pin to specific hashes and it's a change we plan on doing with other dependencies as well.
In any case, sounds good.

Reply:
Done.